server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -178,6 +178,10 @@ public synchronized void removeItem(String key) {
             MTREE_SNAPSHOT_THREAD_CHECK_TIME, TimeUnit.SECONDS);
   }
 
+  /**
+   * we should not use this function in other place,
```

Comment:
Other place is where?  May be you miss something after ','

Reply:
yes, I will add the comments